### DISCUSSION TITLE
archive.rdf4j.org disappeared

### DISCUSSION THREAD

**Discussion by rwalkerands** [eclipse-rdf4j/rdf4j#4650] [General]
Oops, archive.rdf4j.org has disappeared from DNS.

**Comment by rwalkerands** [comment]
Hello?

**Reply by hmottestad** [reply]
@rwalkerands I was not aware that we had an archive. I can see a mention of it in a [stackoverflow post](https://stackoverflow.com/questions/64112724/how-to-increase-max-header-size-for-graphdb-standalone-server), so I can see that it used to work at some point at least. I'll ask on the internal dev channel and also check with the Eclipse Foundation since it's them who are running our infrastructure.

**Reply by hmottestad** [reply]
I've found the Github repo that hosts the webpages for archive.rdf4j.org, https://github.com/Swirrl/archive-sesame .

While we wait for the Eclipse Foundation to help out with the DNS record you can use my fork here: https://hasmac-as.github.io/archive-sesame/

**Comment by rwalkerands** [comment]
See the "announcement" at https://groups.google.com/g/rdf4j-users/c/uNSN6czr-f8/m/sVxBFVKTAwAJ .

(I happen to look after a "project that has not yet migrated" ....)

**Reply by abrokenjester** [reply]
Need a hand migrating? I appreciate it's not always easy to find the resources, but it's a bit of a risk running on software that hasn't had a security fix since 2015...

**Reply by rwalkerands** [reply]
"It's complicated." The service currently operates in "business-as-usual" mode, and I can only make minor alterations.

**Comment by hmottestad** [comment]
I've found the Github repo that hosts the webpages for archive.rdf4j.org, https://github.com/Swirrl/archive-sesame .

While we wait for the Eclipse Foundation to help out with the DNS record you can use my fork here: https://hasmac-as.github.io/archive-sesame/
